### DISCUSSION TITLE
New to Anki Algorithm Video -- Settings Explanation

### DISCUSSION THREAD

**Discussion by MatthewJMullins** [st3v3nmw/obsidian-spaced-repetition#185] [General]
Many of Anki's options are reflected in this plugin. Being new to Anki's algorithm, upon which this plugin is based, I looked into it. 

[This video](https://www.youtube.com/watch?v=lz60qTP2Gx0&t=1990s) steps through all of the options and explains what they all mean from an Anki perspective.

The one thing I noticed missing is that each Anki deck can have its individual settings. Where, in this plugin, the settings apply to all your "decks". Not sure if that's on the roadmap.

Here is [another video](https://youtu.be/uLfczzq9z_8) and at 20:18 he talks about "Ease Hell". Basically, if you tank your ease factor right off the get go, you see that card too many times. The way he over comes this is to set the initial intervals. This is also something to consider adding to this plugin.

**Comment by st3v3nmw** [comment]
> The one thing I noticed missing is that each Anki deck can have its individual settings. Where, in this plugin, the settings apply to all your "decks". Not sure if that's on the roadmap.

Could be done, the user only has to take care of syncing across many devices.

> Here is [another video](https://youtu.be/uLfczzq9z_8) and at 20:18 he talks about "Ease Hell". Basically, if you tank your ease factor right off the get go, you see that card too many times. The way he over comes this is to set the initial intervals. This is also something to consider adding to this plugin.

This is part of a larger issue of how to add new syntax without cluttering up the notes. The primary "problem" is how this plugin, & by extension Obsidian, works. For instance, we can add the learning stage to the syntax we'll have sth like `<!--SR:due-date,interval,ease,learning-stage-->` .  For cloze cards, it's much worse. This was also mentioned in part [here](https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/146).

A potential solution would be to store the scheduling information in a centralized/dedicated file (not in the notes themselves). But now sync conflicts become a huge problem especially for people whose syncing methods don't provide version control & basic merge conflict resolution like what git does. Keep in mind that this is a file which has to be syntactically correct otherwise the plugin won't work. Aaaand, there's been some pushback from the community. So, we're at crossroads.

For more on what has been discussed on ease hell before, please see [this](https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/120) discussion.

**Reply by MatthewJMullins** [reply]
Yeah that makes a lot of sense; didn't realize how many more character that causes. Maybe  There is indeed that "clutter" factor sometimes. I'm also not really sure what to do about that.
